.gitignore

```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: 2022 Tezos Commons
 # SPDX-License-Identifier: LicenseRef-MIT-TC
 
-.stack-work/
+/.stack-work
```

Comment:
I tend to symlink `.stack-work` to `tmpfs`, otherwise I can brew myself a cup of coffee _and_ drink it while the executable is linking :sweat_smile: Trailing `/` makes it so symlink is considered a valid target by git, which is a bit annoying.